saga/sagalogs/file_saga_log.go

```diff
@@ -0,0 +1,387 @@
+package sagalogs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/scootdev/scoot/saga"
+)
+
+// Writes Saga Log to file system.  Not durable beyond machine failure
+// Sagas are stored in a directory.  Each saga has a corresponding directory
```

Comment:
would implementation be simpler if we  used json format for the log contents?  - avoid the formatting and parsing in the code below?